Formula/avimetaedit.rb

```diff
@@ -0,0 +1,18 @@
+class Avimetaedit < Formula
+  desc "A tool that supports embedding, validating, and exporting of metadata in AVI (Standard and OpenDML) files"
+  homepage "https://mediaarea.net/AVIMetaEdit"
+  url "https://mediaarea.net/download/binary/avimetaedit/1.0.1/AVIMetaEdit_CLI_1.0.1_GNU_FromSource.tar.bz2"
+  version "1.0.1"
+  sha256 "3aa4ca6aed51c71684f3fd1bd3b4defc2f9e50a8bd0bdf10e55e86c8ba24aafd"
+
+  def install
+    cd "Project/GNU/CLI" do
+      system "./configure",  "--disable-debug", "--prefix=#{prefix}"
+      system "make", "install"
+    end
+  end
+
+  test do
+    pipe_output("#{bin}/avimetaedit", "--Help")
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software.